lib/application.js

```diff
@@ -491,17 +492,18 @@ const mixin = (application) => {
     }
 
     const servers = config.servers;
-    let server, serverName, serviceName, cpus, i, ilen, srv, port;
     config.servers = {};
-    for (serverName in servers) {
-      server = servers[serverName];
+    for (const serverName in servers) {
+      const server = servers[serverName];
+      let serviceName;
```

Comment:
This declaration should probably be moved to line 508 since it is not used anywhere else, you can also use `const` there.

Reply:
I did as you wanted, but the line 508 has the maximum line length of 80.

Reply:
```js
    const serviceName =
      serverName === "master" ? serverName : serverName + port;
```